grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -186,13 +187,14 @@ class ClientBatchJob {
 
         ClientBatchJob build() {
             final jobParameters = [
-                "timestamp"          : System.currentTimeMillis() as String,
-                "${PATH}"            : pathConfiguration.path,
-                "${HOST}"            : serverBuilder.host,
-                "${PORT}"            : serverBuilder.port,
-                "${USERNAME}"        : credentialsBuilder.username,
-                "${PASSWORD}"        : credentialsBuilder.password,
-                "${WORKFLOW_CONFIGS}": pathConfiguration.workflowConfigIds.join("|")
+                    "timestamp"           : System.currentTimeMillis() as String,
+                    "${PATH}"             : pathConfiguration.path,
+                    "${HOST}"             : serverBuilder.host,
+                    "${PORT}"             : serverBuilder.port,
+                    "${USERNAME}"         : credentialsBuilder.username,
+                    "${PASSWORD}"         : credentialsBuilder.password,
+                    "${EXCLUDE_PATHS}"    : pathConfiguration.excludePaths.join("<sp>"),
```

Comment:
This may have been already discussed, but it seems odd that the join is using `<sp>` and not like `|` as done below.